Capri.Services/Students/StudentDeleter.cs

```diff
@@ -42,7 +42,8 @@ await _context
                     $"Student with id {id} does not exist");
             }
 
-            return await Delete(student);
+            var deleteResult = await Delete(student);
+            return deleteResult;
```

Comment:
Here you can just `return Delete()`
Same situation I've told you about `return await` ;)